bib/cmd/bootc-image-builder/main.go

```diff
@@ -114,6 +131,13 @@ func makeManifest(c *ManifestConfig, cacheRoot string) (manifest.OSBuildManifest
 			return nil, fmt.Errorf("failed to pull container image: %w", err)
 		}
 	}
+	cntSize, err := getContainerSize(c.Imgref)
+	if err != nil {
+		return nil, fmt.Errorf("cannot get container size: %w", err)
+	}
+	c.Filesystems = []blueprint.FilesystemCustomization{
+		{Mountpoint: "/", MinSize: cntSize * 2},
```

Comment:
One other quick thing here, also something like:

```suggestion
	        // As a baseline heuristic we double the size of the input container to
	        // support in-place updates.  This is planned to be more configurable in the future.
		{Mountpoint: "/", MinSize: cntSize * 2},
```

or so?

Reply:
Or perhaps even better a `const containerSizeToDiskSizeMultiplier = 2` at the top with a comment like that?

Reply:
Another great point, fixed.